src/Sarif.Multitool.FunctionalTests/Update-Expected.ps1

```diff
@@ -1,28 +1,38 @@
 ﻿<#
 .SYNOPSIS
-Update the expected output files for the SarifCli functional tests.
+Update the expected output files for the Sarif.Multitool functional tests.
 
 .DESCRIPTION
-This script runs "SarifCli validate" on each input log file in the SarifCli
+This script runs "Sarif.Multitool validate" on each input log file in the SarifCli
 functional tests TestData directory. For each file foo.sarif, it produces
```

Comment:
>foo [](start = 51, length = 3)

Can you change this use of the term 'foo' to something else? 'example.sarif' or *.sarif

Reply:
Fixed. Changed to "example" as you suggest.

---
In reply to: [205215091](https://github.com/Microsoft/sarif-sdk/pull/968#discussion_r205215091) [](ancestors = 205215091)